### PR TITLE
Adding jet PFO information to tree

### PR DESCRIPTION
**Pull request by rbarrue**
Adding PFO-related information for PFlow jets - SumPtChargedPFOPt500PV and fCharged, as well as adding the necessary configuration string in TreeAlgo ("chargedPFOPV").

Tagging @jbossios.

### CODE REVIEW

**Comment by rbarrue** [comment]
I followed the standards used to include other jet variables, as well as the fCharged definition in JetCalibTools: SumPtChargedPFOPt500PV/ pt @ constituent scale.

When I then go look at the actual fCharged distributions from this implementation, I get jets that have values of fCharged > 1. Before I start looking into the reconstruction-level reason this might happen, I want to determine if there's anything wrong with my implementation.

**Comment by jbossios** [comment]
The implementation looks good and fCharged can be sometimes larger than 1.

File: Root/JetContainer.cxx [hunk 1]
```diff
@@ -2646,6 +2686,22 @@ void JetContainer::FillJet( const xAOD::IParticle* particle, const xAOD::Vertex*
...
       } // trackPV
 
+      
+      if ( m_infoSwitch.m_chargedPFOPV && pvLocation >= 0) {
+
+        if ( sumPtChargedPFO500.isAvailable( *jet ) ) {
+          m_SumPtChargedPFOPt500PV->push_back( sumPtChargedPFO500( *jet)[pvLocation] );
+
+          xAOD::JetFourMom_t jetconstitP4 = jet->jetP4("JetConstitScaleMomentum");
+          m_fCharged->push_back( sumPtChargedPFO500( *jet)[pvLocation] / jetconstitP4.Pt());
```
Comment (jbossios) [inline Root/JetContainer.cxx:2696]: @rbarrue could you please create a new PR adding a protection for cases where the jet pt is zero? if jetconstitP4.Pt() is zero we should simply fill -999

Comment (rbarrue) [reply Root/JetContainer.cxx:2696]: Done in https://github.com/UCATLAS/xAODAnaHelpers/pull/1498/

**Pull request merged**